src/cards/ha-weather-card.html

```diff
@@ -48,17 +51,17 @@
           <div class='cond condtemp'>[[attr.temperature]]°</div>
           <div class='cond conddetails'>
             <div class="condsmall" hidden$="[[!attr.wind_speed]]">
-              Wind:
+              [[localize('card.weather.wind')]]:
```

Comment:
Since these are keys for component state attributes, let's not put these under `card`. How about: `attribute.weather.wind_speed`, `attribute.weather.humidity`, etc.

I'd also propose we have this read "Wind speed" to match the attribute name, with the understanding that bearing can follow wind speed without a dedicated label, but that still makes sense to read.